### PR TITLE
Fix logout function, dynamically update name in navbar

### PR DESCRIPTION
**Pull request by leogoldberg**
Logout function was not working because apiClient.config.headers was undefined. Fixed this and confirmed its working now
Added name field to login response data and update this dynamicallly in the navbar

### CODE REVIEW

File: src/services/EventService.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 
-import axios  from 'axios';
+import axios from 'axios';
 
 const TOKEN_KEY = 'JWT';
-const URL = `http://35.202.57.20:${process.env.PORT || 5000}`;
+const URL = `http://localhost:${process.env.PORT || 5000}`;
```
Comment (jorawarc) [inline src/services/EventService.js]: This is just for testing?

File: src/services/EventService.js [hunk 2]
```diff
@@ -20,26 +20,25 @@ apiClient.interceptors.request.use(function (config){
 const userLogin = (payload, callback) => {
     apiClient.post('/auth/login', payload, callback)
         .then((res) => {
-            localStorage.setItem(TOKEN_KEY, res.data);
-            callback(true);
+            localStorage.setItem(TOKEN_KEY, res.data.token);
+            callback(true, res.data.name);
```
Comment (jorawarc) [inline src/services/EventService.js:24]: The server should only be returning a `Bearer <token>` on a successful authentication. I don't think the `.token` and `.name` are accessible methods. Were you about to confirm the behaviour locally?

File: src/services/EventService.js [hunk 3]
```diff
@@ -20,26 +20,25 @@ apiClient.interceptors.request.use(function (config){
...
         }).catch(error => {
             console.log(error);
             callback(false);
-    });
+        });
 }
 
 const userLogout = () => {
-    delete apiClient.config.headers.authorization;
```
Comment (jorawarc) [inline src/services/EventService.js:32]: Gonna need an explanation of why we can delete this line

File: src/services/EventService.js (hunk 2 above)
Comment (jorawarc) [reply src/services/EventService.js:24]: Oh I see, [Backend-PR#6](https://github.com/c9y7ip/ezpark-backend/pull/6) handles this

File: src/services/EventService.js (hunk 3 above)
Comment (JasonXD-CS) [reply src/services/EventService.js:32]: apiClient is not persisting the authorization header. It's persisted in localStorage so removing from localStorage is the correct way to do it

File: src/services/EventService.js (hunk 1 above)
Comment (leogoldberg) [reply src/services/EventService.js]: Yep my bad I'll revert this back

**Pull request merged**